addon/styles/addon.scss

```diff
@@ -10,11 +17,13 @@
   user-select: none;
 
```

Comment:
Are these initial styles the primary styles? If so, they should still be scoped to .primary